packages/app/src/specs/SpecsList.vue

```diff
@@ -487,6 +489,15 @@ function refreshPage () {
   location.reload()
 }
 
+const handleLoggedin = () => {
```

Comment:
Why is the `location.reload` only necessary on the `SpecsList` page? You can log in on the runs page that has cloud data, does that not need a refresh?

Reply:
This is a good point and I don't really know the answer - I believe this was added with the first pass at the ACI work. I'd say this should be checked more thoroughly and maybe we are missing something - or maybe we don't really need the refresh. Logging in from the header on the Specs List page does _not_ trigger a refresh, only logging in from the tooltip does, which doesn't make sense if the refresh is needed.

I think next sprint we should consolidate Login and Cloud Connect into a single top-level component in App opened/closed via a store, that behaves the same way no matter where you triggered it from. I'm going to make that ticket and I'll add this "confirm when and why we need to reload the page for the user" point & probably get with @mike-plummer about the context there.

Reply:
@marktnoonan @ZachJW34 I don't have a firm answer to why this was initially added - my suspicion is that either `SpecHeaderCloudDataTooltip.vue` or `SpecsList.vue` needs a `cloudViewerChange` subscription (see `CloudConnectModals.vue`) so that the tooltip state is properly updated after login. This `reload` is very likely a workaround to get data looking "correct" by forcing a larger re-query

Reply:
oh cool. when moving this to frontend-shared we can perhaps get the subscription wrapped up in the overall login process, or at least lifted up in App such that it would trigger from logging in on the runs page via the header, etc.